CHANGES.md

```diff
@@ -1,3 +1,8 @@
+## In the next release...
+
+* **Breaking Change**: Modified `io.l5d.mesh` and `io.l5d.thriftNameInterpreter` default serve addresses from 0.0.0.0 to 127.0.0.1.
+* **Breaking Change**: Modified linkerd admin and namerd admin default serve addresses from 0.0.0.0 to 127.0.0.1.
```

Comment:
wording nit.  clearer to me as:

"modified _ to serve on 127.0.0.1 by default (instead of 0.0.0.0)."